mcos/optimizer/__init__.py

```diff
@@ -0,0 +1,2 @@
+from mcos.optimizer.base import AbstractOptimizer
+from mcos.optimizer.markowitz import MarkowitzOptimizer
```

Comment:
This allows for imports like
```
from mcos.optimizer import MarkowitzOptimizer
```
instead of
```
from mcos.optimizer.markowitz import MarkowitzOptimizer
```